File: mlb_fan_highlights/src/javasings.py
import requests

url = "http://localhost:8080/api/v1/podcast"
payload = {
    "team": "Dodgers",
    "players": ["Shohei Ohtani", "Mookie Betts", "Freddie Freeman"],
    "timeframe": "Last game",
    "game_type": "Regular Season",
    "language": "english",
    "opponent": "Giants"
}

response = requests.post(url, json=payload)
print(response.status_code)
print(response.json())

File: mlb_fan_highlights/src/javasings.js
fetch('https://mlb-strings-1011675918473.us-central1.run.app/api/v1/podcast', {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json',
    },
    body: JSON.stringify({
      team: "Dodgers",
      players: ["Shohei Ohtani", "Mookie Betts", "Freddie Freeman"],
      timeframe: "Last game",
      game_type: "Regular Season",
      language: "english",
      opponent: "Giants"
    }),
  })
  .then(response => response.json())
  .then(data => console.log(data))
  .catch((error) => console.error('Error:', error));